src/style.css

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css2?family=Della+Respira&display=swap');
+
+:root {
+  --primary-color: #717754;
+}
+
+* {
+  font-family: 'Della Respira', serif;
+  box-sizing: border-box;
+}
+
+html {
+  color-scheme: dark;
+}
+
+h1 {
+  font-size: 55px;
+}
+
+section {
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  margin: 25px 0;
+  font-size: 1em;
+  min-width: 300px;
+  padding: 25px;
+  box-shadow: 7px 7px 21px #333, -7px -7px 21px #222;
+  border-radius: 20px;
+  overflow: hidden;
+}
+
+.styled-table {
+  border-collapse: collapse;
+}
+
+.styled-table thead tr {
+  background-color: var(--primary-color);
+  color: #fff;
+}
+
+.styled-table th,
+.styled-table td {
+  padding: 15px 25px;
+}
+
+.styled-table tbody tr {
+  border-bottom: 1px solid #ddd;
+}
+
+.styled-table tbody tr:nth-of-type(even) {
+  /* background-color: #f3f3f3; */
+}
+
+.styled-table tbody tr:last-of-type {
+  border-bottom: 2px solid var(--primary-color);
+}
+
+form {
+  padding: 50px 5px;
+}
+
+@media (min-width: 768px) {
+  section {
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 50px;
+  }
+}
+
+/* utility class */
+.text-center {
+  text-align: center;
+}
+
+.flex-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.space-between {
+  justify-content: space-between;
+}
```

Comment:
Nice job so far👍👍. You have an excellent user interface😉. but I think we can make it better by sticking to how the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) was structured😉. 

 See the wire Wireframe
<img width="484" alt="image" src="https://github.com/ichsansandy/leaderboard-microverse/assets/64643995/ff8209e7-c8ff-4d0e-bde6-7184b662691a">

From the above, you can see all the contents are fitted to the screen😉 Your input fields look great but I want you to take note of the scores section😉 The width of your score section seems too large and occupies almost all the width of the screen which causes the page to overflow, leading to a poor user experneice🥲. Please let's kindly fix that. Try to make sure the score section takes about 50% of the screen width as shown in the wireframe👍.

<img width="963" alt="image" src="https://github.com/ichsansandy/leaderboard-microverse/assets/64643995/1ccabafc-1c9c-4123-b27c-85b4b7d49332">



https://github.com/ichsansandy/leaderboard-microverse/assets/64643995/ffa31084-243a-45a4-b52c-af4d1be5dd54